minor update in the new user's sudo permissions ..

dimkatsi91 opened the pull request:
New created user belongs to the existing 'wheel' group

dimkatsi91 committed:
minor update in the user sudo permissions ..

cerebrux commented on archon.sh:
```diff
@@ -12,6 +12,12 @@
 
+
+## ADDITION START
+sudoers_file=/etc/sudoers
```
Tip: There is no need to save a value in a variable if it is going to be used once :)

cerebrux commented on archon.sh:
```diff
@@ -218,7 +224,17 @@ function chroot_stage {
+
+	## ADDITION START ?
+	## Since new user belongs to wheel group now (via useradd command), 
+	## just uncomment the appropriate line  from the /etc/sudoers file
+	sed -i "s/# %wheel ALL=(ALL) ALL/%wheel ALL=(ALL) ALL/" $sudoers_file
+	## ADDITION END ?
```
As mentioned before there is no need to use a variable for something used only once. Οπότε εδώ καλύτερα να χρησιμοποιείς απευθείας την /etc/sudoers

cerebrux reviewed (changes requested):
Κάνε τις αλλαγές που πρότεινα στα σχόλια

dimkatsi91 committed:
minor update ..

dimkatsi91 commented:
Hi,

Just added this variable cause I thought You would like to add
a function for the /etc/sudoers file for 'sanity check' via 'visudo -f -c sudoers' command,
just after the sed command, since a false sed upon this file is going to lock wheel group users for super-user priviledged commands.

Anyway, I just removed this $sudoers_file variable from my forked repo archon.sh script
in develop branch.

Best Regards,
Dimosthenis Katsimardos

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
On Monday, 4 May 2020 6:51 PM, Salih Emin <notifications@github.com> wrote:

> @cerebrux requested changes on this pull request.
>
> Κάνε τις αλλαγές που πρότεινα στα σχόλια
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, [view it on GitHub](https://github.com/CerebruxCode/Archon/pull/57#pullrequestreview-405117259), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AGMVEWYCVQLLFZNQOVJI24LRP3P6ZANCNFSM4MYJ7CUA).
